### PR TITLE
fix: re-add background to tertiary button

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Tertiary buttons have the same background hover color as some of the elements they are laid on top of. Given there is no opacity in their color it looks like the hover state is broken. For example:
Screenshot 2023-11-27 at 14 45 34

## Changes

- Reuses the `glass-bg-3000` color as the background given its opacity fits the use case pretty well
- Tweaked the dark mode color & opacity to make the contrast a little better while retaining the breadcrumbs background color as much as possible

|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 00 44 | Screenshot 2023-11-27 at 15 00 51 |

|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 05 23 | Screenshot 2023-11-27 at 15 05 30 |

|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 06 10 | Screenshot 2023-11-27 at 15 06 02 |

### Particularly bad in dark mode
|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 11 12 | Screenshot 2023-11-27 at 15 11 20 |

|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 12 46 | Screenshot 2023-11-27 at 15 12 53 |

### Glass header difference is almost unnoticeable
|Before|After|
|----|----|
| Screenshot 2023-11-27 at 15 12 05 | Screenshot 2023-11-27 at 15 12 10 |

## How did you test this code?

Snapshots

### CODE REVIEW

File: frontend/src/styles/vars.scss [hunk 1]
```diff
@@ -125,7 +125,6 @@ $colors: (
     'secondary-3000-hover-light': #cfd1c2,
     'accent-3000-light': #eeefe9,
     'bg-3000-light': #f3f4ef,
-    'bg-hover-3000-light': #f3f4ef,
```
Comment (daibhin) [inline frontend/src/styles/vars.scss:128]: Seemed to be unused

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss [hunk 2]
```diff
@@ -239,7 +239,7 @@
 
             &:not([aria-disabled='true']):hover,
             &.LemonButton--active {
-                background-color: var(--bg-3000);
+                background-color: var(--glass-border-3000);
```
Comment (benjackwhite) [inline frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:242]: At some point we probably want to change the name of that var

**Review by benjackwhite — APPROVED** [review]
Simple!

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss (hunk 2 above)
Comment (daibhin) [reply frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:242]: I did consider that. The breadcrumbs and this button are the only places it's used right now so should be relatively easy to find a suitable name once we get a sense of where else it is needed

File: frontend/__snapshots__/lemon-ui-lemon-button--active--light.png
Comment (daibhin) [inline frontend/__snapshots__/lemon-ui-lemon-button--active--light.png:1]: Snapshots helping out!!

**Pull request merged**